FFXIVClientStructs/FFXIV/Client/Game/InventoryManager.cs

```diff
@@ -123,11 +195,17 @@ public enum InventoryType : uint {
 
     Currency = 2000,
     Crystals = 2001,
+    MailEdit = 2002, // used by LetterEditor
     Mail = 2003,
     KeyItems = 2004,
     HandIn = 2005,
+    Unknown2006 = 2006,
     DamagedGear = 2007,
+    Unknown2008 = 2008,
```

Comment:
```suggestion
    DamagedGear = 2007,
```
I don't think you meant to commit those

Reply:
Yes, I did intend to commit them, becuase they are valid ids in the games array.
Like I said, they are probably for trading and reparing with other players.
Besides them and the 9999, there is also a 2000 and 2013 probably used for start and end marking of these types, commonly found in enums.
Whenever the correct purpose is found, they can be marked obsolete and point to the correct name. 🙂

Reply:
Yeah, sorry I should go to bed and review when awake actually